clone.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+echo "cloning git repo"
+git clone https://${GITHUB_TOKEN}@github.com/${GITHUB_USER}/${GITHUB_REPO}.git target
```

Comment:
git author is already set in the Dockerfile,

We should check out the correct branch yes